src/jni/platforms/jvm_default.cpp

```diff
@@ -16,7 +16,7 @@ namespace jni {
 
     JavaVM* Jvm::vm = nullptr;
     jint Jvm::version = 0;
-    Jvm::Type Jvm::vm_type {Jvm::JVM};
+    Jvm::Type Jvm::vm_type {Jvm::NONE};

```

Comment:
Why is it NONE here ?

Reply:
If it was a bitmask, it would mean that a VM can be ART and Graal at the same time (for example).
The idea behind none is because I'm using VARIANT_ENUM_CAST so godot can read the enum as Variant value, which make the code easier in some parts, and the 0 value for Variant enum is basically the default for Godot enums (most but not all enums and Godot have a NONE value as the 0).
I know use for the json and command line as the "auto" tag, so the VM will automatically switch to ART on Android and graal on IOS.
Otherwise, with the way I know export the json, it would always be generated with JVM by default. And even if it's not incorrect for desktop anyway, i prefer the automatique mode to be explicit.